.github/workflows/main.yml

```
@@ -0,0 +1,37 @@
+name: Docurium
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [head, 2.6, 2.5, 2.4]
```

While we're updating things, this should probably be
```suggestion
        ruby: [head, 3.0, 2.7, 2.6]
```

as that's the supported versions upstream

---

👍